README.md

```diff
@@ -14,19 +14,19 @@ This project requires a [Docker](https://www.docker.com) installation.
 ## Installation and Execution ##
 
 - Build the docker image:
-  - `docker-compose build`
+  - `docker compose build`
 - Change the credentials in the following configuration files:
   - `secrets/admiral.yml`
   - `secrets/redis.conf`
-  - `docker-compose.yml`
+  - `compose.yaml`
```

Comment:
Today I learned that [`compose.yaml` is preferred to `compose.yml`](https://docs.docker.com/compose/intro/compose-application-model/#the-compose-file).  Since that is the case, we should change the other file extensions listed in this README as well (e.g. `admiral.yml` -> `admiral.yaml`, etc.).

Reply:
See [aa6d857](https://github.com/cisagov/admiral/pull/79/commits/aa6d8570e783e3748d555b203828cae00d2a95e0).

Reply:
Looks good, thanks!